quantum/process_keycode/process_ucis.c

```diff
@@ -118,7 +122,6 @@ bool process_ucis (uint16_t keycode, keyrecord_t *record) {
       return false;
     }
   }
```

Comment:
```suggestion
  }

```
This suggestion readds the newline.